at-chuongphu/excercise5.rb

```diff
@@ -0,0 +1,41 @@
+# ex1
+class Student
+	attr_accessor :first_name, :last_name, :list_course
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/1c0a5adcb251976aac9f1128795ec3dbcba91c88/at-chuongphu/excercise5.rb#L3"></span>Layout/IndentationWidth: Use 2 (not 1) spaces for indentation. (https://github.com/rubocop-hq/ruby-style-guide#spaces-indentation)</td>
    </tr>
  </tbody>
</table>